02-intermediate-ruby/object-inheritance.md

```diff
@@ -130,6 +132,8 @@ Note: Now that we are adding a new attribute to the `Apartment` class, we are re
 
 `super` **must** be the first line in a subclass' `initialize` method.  If `super` is called without an argument, it will use the parameters from the subclass' `initialize` method.
 
+> **Exercise**: Update the `Circle` class to have a `radius` attribute.
```

Comment:
Similar to the cmoment above; this is right underneath talking about `super`. Is there a more specific direction that will give students immediate feedback on if they're doing the right thing, whether it's code to run, or sample code to check?

Reply:
Yes, great point. I'll add instructions for that. Thanks!